assets.src/src/redmine-tracky/controllers/timer-controller.ts

```diff
@@ -56,18 +55,28 @@ export default class extends Controller {
         }
     }
 
-    private timeDifference(): string {
-        const startDateTime = this.convertToDateTime((this.startTarget as HTMLInputElement).value);
-        const endDateTime = this.convertToDateTime((this.endTarget as HTMLInputElement).value);
+    private timeDiffToString(timeDiff: TimeDiff) {
+        const sign = (timeDiff.minutes < 0 || timeDiff.seconds < 0) ? '-' : '';
```

Comment:
i sometimes think, it's not better to have less lines but sacrifice that the code stays readable. that's why i extracted a lot of things. like the `removeHoursIfZero` function. extraction is a good pattern to comment your code:

```suggestion
        const negativeDuration: (timeDiff.minutes < 0 || timeDiff.seconds < 0);
        const sign =  negativeDuration ? '-' : '';
```